### PR TITLE
make 'gas' for erc1155_deploy and tac_control_contract configurable

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

make 'gas' amount configurable, for contract deployment and calls in skill `erc1155_deploy`.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
n/a

### CODE REVIEW

File: tests/test_packages/test_contracts/test_erc1155/test_contract.py [hunk 1]
```diff
@@ -349,6 +349,7 @@ def sign_send_verify_deploy_init_transaction(
             ]
         )
 
+    @pytest.skip
```
Comment (marcofavorito) [inline tests/test_packages/test_contracts/test_erc1155/test_contract.py:352]: @DavidMinarsch

File: tests/test_crypto/test_ethereum.py [hunk 2]
```diff
@@ -204,6 +204,7 @@ def test_construct_sign_and_submit_transfer_transaction(
     assert tx != transaction_receipt, "Should not be same!"
 
 
+@pytest.skip
```
Comment (marcofavorito) [inline tests/test_crypto/test_ethereum.py:207]: @DavidMinarsch faucet not working on Ganache

Comment (DavidMinarsch) [reply tests/test_crypto/test_ethereum.py:207]: Ok, let's open an issue

File: tests/test_packages/test_contracts/test_erc1155/test_contract.py (hunk 1 above)
Comment (DavidMinarsch) [reply tests/test_packages/test_contracts/test_erc1155/test_contract.py:352]: Please open issue too

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**